### DISCUSSION TITLE
reinterview node fails immediatly after response

### DISCUSSION THREAD

**Discussion by TomMalfrere** [zwave-js/zwave-js#7526] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
reinterview node 072 fails every time
Device is declared dead while it was just communicating...
I see the interview doing a node ping which fails and when I do a manual ping it passes... (timeout to low?)
node 072 is the most remote node in my network...

What did you expect to happen instead?
retry maybe...

  click "reinterview node" on device 072

Anything else we should know?
I was using the latest zwave-js-ui docker but went back to zwavejs/zwave-js-ui:9.26.0
I had the issues in both,

### Software versions

docker with zwavejs/zwave-js-ui:9.26.0

Driver (node-zwave-js): 13.10.3

Z-Wave JS UI: 9.26.0.47f4c89

sends to mosquitto (latest docker om docker hub) and then to domoticz

### Z-Wave Controller (Stick/Dongle/...)

Aeotec z-stick gen 5 
FW: v1.0 SDK: v6.51.2

### Device information

Manufacturer: Neo
Model name: wallplug NAS-WR01ZE 
Node ID: 072

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-12-31.log](https://github.com/user-attachments/files/18282125/zwavejs_2024-12-31.log)

**Comment by AlCalzone** [comment]
You have too much energy reporting, approximately one report every 1.5 seconds:
https://zwave-js.github.io/node-zwave-js/#/troubleshooting/first-steps?id=reduce-reports-from-the-network
